src/input_context/actions.rs

```diff
@@ -24,8 +25,8 @@ use crate::{
 #[derive(Component)]
 pub struct Actions<C: InputContext> {
     gamepad: GamepadDevice,
-    bindings: Vec<ActionBinding>,
-    action_map: TypeIdMap<UntypedAction>,
+    pub bindings: Vec<ActionBinding>,
+    pub action_map: TypeIdMap<UntypedAction>,
```

Comment:
These two fields needs to be kept in sync, so we need to encapsulate them. I played with the API for it locally a little and I'd suggest the following:
- Remove `C` from `NoActionError`. I think it will be obvious for the user from which context the error is from.
- Then move all fields except `marker` from `Actions<C>` to `UntypedActions`.
- Store `UntypedActions` inside `Actions<C>` and deref to `UntypedActions`.

Maybe as a separate PR to simplify this one? :thinking: